.github/workflows/integration-tests-amd.yml

```diff
@@ -115,6 +115,9 @@ jobs:
           if [ ! -d "${INSTRUMENTATION_LIB_DIR}" ]; then
             echo "Could not find '${INSTRUMENTATION_LIB_DIR}'" ; exit -1
           fi
+          pytest -rfs python/test/gluon/test_frontend.py
```

Comment:
You can just do `pytest --capture=tee-sys -rfs -n 8 python/test/gluon/` to run over the full directory so that later if there are new tests added we can automically picking them up.